HHS/OPRE-OPS
# Footer fixes

Getting the global footer to be a little less placeholder-ish and closer to some semblance of what it will need to be  and adding the mandated log in disclaimer

## What changed

* Adds the HHS/ACF-mandated footer. For now it's as a forced modal before you can log in, but we can change the UX later to be whatever is best. 
* Removes broken footer links
* Added more-relevant footer links and removed irrelevant ones
* Removed placeholder agency contact info pending further instructions

## Issue

#651 

## How to test

fire up the frontend container or do a `bun run start` on the frontend and check out the initial landing page

## Screenshots

<img width="1072" alt="Screenshot 2024-06-12 at 20 07 52" src="https://github.com/HHS/OPRE-OPS/assets/928984/c2318012-f846-4aa1-ad7f-3bfa8907e591">

## Definition of Done Checklist


## Links
